docs/Developers/Development_Setup.rst

```diff
@@ -91,32 +91,27 @@ https://gitlab.com/coala/coala-utils/
 Installing from Git
 -------------------
 
-We recommend first installing the latest development snapshot of coala's
-master branch from and all of its dependencies with pip3 using
+We recommend installing coala from the master branch for latest updates
+and its dependencies with pip3 using the commands given below. The
+``-e`` tag installs the project in the editable mode from the given path.
 
 ::
 
     (coala-venv)$ git clone https://github.com/coala/coala
     (coala-venv)$ cd coala
-    (coala-venv)$ pip3 install .
+    # The next line installs coala in editable mode.
+    (coala-venv)$ pip3 install -e .
     (coala-venv)$ cd ..
     (coala-venv)$ git clone https://github.com/coala/coala-bears
     (coala-venv)$ cd coala-bears
-    (coala-venv)$ pip3 install .
-
-Once you have forked the repository you would like to modify, you can
-install a repository-backed version of the repository using
-
-::
-
-    (coala-venv)$ pip3 install -e <path/to/forked/repository>
+    # The next line installs coala-bears in editable mode.
```

Comment:
this too, add `We recommend installing coala and coala bears from...`

Reply:
since you have already described earlier what `-e` does

Reply:
I think we should keep the line about recommend.